ts/src/test/test.ts

```diff
@@ -206,9 +207,20 @@ export default class testMainClass extends baseMainTestClass {
         const exchange = initExchange (exchangeId, exchangeArgs);
         await this.importFiles (exchange);
         this.expandSettings (exchange, symbol);
+        this.checkIfSpecificTestIsChosen ();
         await this.startTest (exchange, symbol);
     }
 
+    checkIfSpecificTestIsChosen () {
```

Comment:
@ttodua isn't easier to accept the name in the symbol argument so you can do `test binance fetchBalance` ? I'm doing something similar in the static tests

Reply:
I refrained to do that, because the order of argvs matter when calling run-tests, so to avoid possible confusions betweem run-tests vs test.ts/py, I've thought this way might be better, also we could accept several method names like `--fetchBalance --fetchDeposits`... but as you tell, i'll do so.

Reply:
bu in this case you're relying on the order as well, because we also provide the --request/--response flags

Reply:
how do we rely on the order here? we could provide flags with mixed order like:
`--request --fetchBalance --createOrder`... etc , not?

Reply:
ok, I think i can do as you say, i'll add also array support as the second argument

Reply:
done, now it can be run :
```
.... test.ts binance fetchDeposits,fetchWithdrawals  
```
(accepts single or comma separated)